src/editor/index.php

```diff
@@ -118,7 +132,8 @@ function simplifyPath( $path ) {
 }
 
 // .. as a fully qualified filename
-$strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+// $strSchemaFQFilename = realpath( simplifyPath( $strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename) );
+$strSchemaFQFilename = realpath($strConfigPath . ($isDemo?'demo/':'') . $strSchemaFilename );
```

Comment:
realpath works on filesystem paths, simplifyPath only on the string that represents a path.

When it comes to symlinks it makes a huge difference. This is relevant when the web root contains symlinks as then the ".." in a URL will collide with a ".." in the file system.
Leaving it out will cause file-not-found as a best and a security leak as a worst case.